src/utils/registration-helpers.ts

```diff
@@ -676,3 +676,9 @@ export const openFileInNewTab = (fileUri: string) => {
     setTimeout(() => window.open(fileUri, '_blank'));
   }
 };
+
+export const compareRegistrationAndValues = (registration: Registration, values: Registration) => {
+  return registration.entityDescription?.publicationDate?.year !== values.entityDescription?.publicationDate?.year
```

Comment:
Nå vil denne dukke opp også for ikke-NVI-kandidater, så legg inn en sjekk for om `registration` i det hele tatt _er_ en NVI-kandidat også. Mener denne bare skal vises om resultatet allerede har en `approval` som er godkjent eller avvist
Burde sikker gjør noe gjenbrukbart for dialogen som dukker opp med advarsel om at brukern kan gjøre endringer som påvirker NVI. Denne dialogen skal aldri vises om den dialogen ikke ble vist i forkant

Reply:
https://github.com/BIBSYSDEV/NVA-Frontend/pull/4771/